src/how-tos/mqtt/mqtt_kafka_broker_zilla.yaml

```diff
@@ -1,20 +1,48 @@
+---
 name: zilla-mqtt-kafka-broker
 bindings:
-
-# Proxy service entrypoint
+  # Proxy service entrypoint
   north_tcp_server:
     type: tcp
     kind: server
     options:
       host: 0.0.0.0
-      port: 7183
-    exit: north_mqtt_server
+      port:
+        - 7114
+        - 7183
+    routes:
+      - when:
+          - port: 7114
+        exit: north_ws_http_server
+      - when:
+          - port: 7183
+        exit: north_mqtt_server
+
+  # WebSocket server
+  north_ws_http_server:
```

Comment:
Lets rename to `north_http_server` and add `upgrade: websocket` to route when headers below.